cmake/OpenCVPCHSupport.cmake

```diff
@@ -16,6 +16,11 @@ IF(CMAKE_COMPILER_IS_GNUCXX)
 
     IF(NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.2.0")
         SET(PCHSupport_FOUND TRUE)
+        IF(${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
```

Comment:
Precompiled headers is compiler/build system feature, not a specific platform. Does OpenBSD use GCC? What is the problem?

Reply:
We build ports with clang and GCC depending on the arch. Thus PCH is a nogo.

Reply:
> depending on the arch

Do you use CMake cross-compiling mode? CMake toolchain? Do you see CMAKE_CROSSCOMPILING variable?

Probably you could turn off [`ENABLE_PRECOMPILED_HEADERS`](https://github.com/opencv/opencv/blob/3.3.1/CMakeLists.txt#L295) in your build scripts.